### PR TITLE
Docs for translateWIKI

### PR DESCRIPTION
**Pull request by BPerlakiH**
Fix: https://github.com/kiwix/apple/issues/561

### CODE REVIEW

File: Support/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -75,12 +75,9 @@
 "flavor_tag.help.no_pic" = "most pictures have been removed";
 "flavor_tag.help.mini" = "only a subset of the text is available, probably the first section";
 
-"grid_section.preview.title" = "Header Text";
-"grid_section.preview.content" = "Content";
 "library_refresh_time.last" = "Just Now";
```
Comment (BPerlakiH) [inline Support/en.lproj/Localizable.strings:78]: the preview ones were not used in production -> no translation needed.

File: Support/en.lproj/Localizable.strings [hunk 2]
```diff
@@ -201,7 +196,6 @@
 "bookmark.toolbar.show_sidebar.label" = "Show Sidebar";
 
 "browser_tab.toolbar.show_sidebar.label" = "Show Sidebar";
-"app.navigation.title" = "Kiwix";
```
Comment (BPerlakiH) [inline Support/en.lproj/Localizable.strings:204]: This is not used anywhere.

File: Support/en.lproj/Localizable.strings [hunk 3]
```diff
@@ -210,7 +204,6 @@
 "search_result.button.clear" = "Clear";
 "search_result.clear_dialog.title" = "Clear Recent Searches";
 "search_result.clear_dialog.button.all" = "Clear All";
-"search_result.clear_dialog.dialog-description" = "All recent search history will be removed.";
```
Comment (BPerlakiH) [inline Support/en.lproj/Localizable.strings:213]: This is a duplicate.

File: Views/BuildingBlocks/GridSection.swift [hunk 4]
```diff
@@ -29,8 +29,8 @@ struct GridSection<Content: View>: View {
 
 struct GridSection_Previews: PreviewProvider {
     static var previews: some View {
```
Comment (BPerlakiH) [inline Views/BuildingBlocks/GridSection.swift:31]: This is a preview section, not production part of the application, no translations needed.

File: Views/BuildingBlocks/Message.swift [hunk 5]
```diff
@@ -26,7 +26,7 @@ struct Message: View {
 
 struct Message_Previews: PreviewProvider {
```
Comment (BPerlakiH) [inline Views/BuildingBlocks/Message.swift:27]: Preview only

File: Views/Library/ZimFileDetail.swift [hunk 6]
```diff
@@ -333,14 +333,14 @@ struct ZimFileDetail_Previews: PreviewProvider {
         zimFile.created = Date()
         zimFile.downloadURL = URL(string: "https://www.example.com")
         zimFile.fileID = UUID()
```
Comment (BPerlakiH) [inline Views/Library/ZimFileDetail.swift:335]: It is also a preview, not production code.

File: Views/Settings/Settings.swift [hunk 7]
```diff
@@ -178,7 +178,7 @@ struct Settings: View {
         } header: {
             Text("library_settings.tab.library.title".localized)
         } footer: {
```
Comment (BPerlakiH) [inline Views/Settings/Settings.swift:180]: This was referencing the wrong key, fixed it.

**Review by rgaudin — APPROVED** [review]
LGTM ; thank you for this effort. I haven't reviewed the strings themselves and am not capable of doing it though.

**Comment by kelson42** [comment]
@BPerlakiH It's that that this PR mixes two different things:
* the string doc in qqq
* the translations from TW

... or I understand something wrongly...

**Comment by kelson42** [comment]
Hmmm… I gotbit wrong. Sorry.

**Pull request merged**